rune/Makefile

```diff
@@ -13,6 +13,8 @@ BUILDTAGS ?= seccomp selinux apparmor
 COMMIT_NO := $(shell git rev-parse HEAD 2> /dev/null || true)
 COMMIT ?= $(if $(shell git status --porcelain --untracked-files=no),"$(COMMIT_NO)-dirty","$(COMMIT_NO)")
 VERSION := $(shell cat ./VERSION)
+INCLAVARE_CONTAINERS_VERSION := $(shell cat ../VERSION)
```

Comment:
Once rune is available from a separate repo, ../ doesn't make sense any more at that time. You need to export INCLAVARE_CONTAINERS_VERSION on top of of Makefile, and use INCLAVARE_CONTAINERS_VERSION ?= to query the version through cat ../VERSION.